src/utils/common.ts

```diff
@@ -229,10 +229,30 @@ export const getSamplesBetween = (min: number, max: number, size: number) => {
   return intervals;
 };
 
+/**
+ * A promise that resolves after a specified number of seconds
+ * @param seconds The number of seconds to wait
+ */
 export async function delay(seconds: number) {
   return new Promise((resolve) => setTimeout(resolve, seconds * 1000));
 }
 
+/**
+ * Converts a string to a hex string
+ * @param input The string to convert
+ * @returns The hex string
+ */
+export function utf8ToHex(input: string) {
+  return ethers.utils.formatBytes32String(input);
+}
```

Comment:
Moved `utf8ToHex(...)` to `utils.ts`

Reply:
Returns type `string<DataHexString<32>>` according to [ethers](https://docs.ethers.org/v5/api/utils/strings/#utils-formatBytes32). Can we get away with a `string` return type annotation?